GramAddict/core/views.py

```diff
@@ -637,6 +673,18 @@ def isPrivateAccount(self):
         )
         return private_profile_view.exists()
 
+    def haveStory(self):
```

Comment:
isStoryAvailable would be a better naming?

Reply:
Renamed